internal/agent/device/console/console_windows.go

```diff
@@ -0,0 +1,59 @@
+//go:build windows
+
+package console
+
+import (
+	"fmt"
+	"golang.org/x/sys/windows"
+	"syscall"
+	"unsafe"
+)
+
+// Define Windows API functions
+var (
+	kernel32                       = syscall.NewLazyDLL("kernel32.dll")
+	procSetConsoleScreenBufferSize = kernel32.NewProc("SetConsoleScreenBufferSize")
+	procSetConsoleWindowInfo       = kernel32.NewProc("SetConsoleWindowInfo")
+)
+
+const (
+	maxConsoleWidth  = 32767
+	maxConsoleHeight = 32767
+)
+
+func init() {
+	// Ensure DLL and procedures are loaded
+	if err := kernel32.Load(); err != nil {
+		panic(fmt.Sprintf("failed to load kernel32.dll: %v", err))
+	}
+}
+
+func setConsoleSize(fd uintptr, size TerminalSize) error {
```

Comment:
Why did you need to do anything with the console at all?

agent code should not be compiled for the CLI, where is the CLI pulling the internal/agent/device/console ?

Reply:
removed all `console` stuff